src/views/nunjucks/returns/processing-upload.njk

```diff
@@ -0,0 +1,10 @@
+{% extends "./nunjucks/layout.njk" %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">{{ pageTitle }}</h1>
+  <p class="govuk-body-l">Please wait. This may take up to a minute.</p>
+  <div style="display: flex; justify-content: center;">
```

Comment:
We should probably avoid inline styles and create a `spinner` component in src/assets/sass/v2/blocks